examples/training_scripts/height_example/train_height.py

```diff
@@ -15,20 +15,46 @@
 """
 import logging
 import time
+from typing import Optional
 
 from syne_tune import Reporter
 from argparse import ArgumentParser
 
+from syne_tune.config_space import randint
+
 
 report = Reporter()
 
 
+RESOURCE_ATTR = "epoch"
+
+METRIC_ATTR = "mean_loss"
+
+METRIC_MODE = "min"
+
+MAX_RESOURCE_ATTR = "steps"
+
+
+def train_height(step: int, width: float, height: float) -> float:
+    return 1.0 / (0.1 + width * step / 100) + 0.1 * height
+
+
+def height_config_space(max_steps: int, sleep_time: Optional[float] = None) -> dict:
+    kwargs = {"sleep_time": sleep_time} if sleep_time is not None else dict()
+    return {
+        MAX_RESOURCE_ATTR: max_steps,
+        "width": randint(0, 20),
+        "height": randint(-100, 100),
+        **kwargs,
+    }
+
+
 if __name__ == "__main__":
     root = logging.getLogger()
     root.setLevel(logging.INFO)
 
     parser = ArgumentParser()
-    parser.add_argument("--steps", type=int)
+    parser.add_argument("--" + MAX_RESOURCE_ATTR, type=int)
```

Comment:
you could use `f"--{MAX_RESOURCE_ATTR}"`. isn't type `str`?

Reply:
MAX_RESOURCE_ATTR is str, but it is just the name, its value is int.